### DISCUSSION TITLE
Plugin "Keys"

### DISCUSSION THREAD

**Discussion by AndreyIvanov** [Heistergand/fanfields2#4] [Ideas]
is it possible to use "keys"together with the plugin? it will be very convenient if you show the presence of keys, and not just the farm

**Comment by Heistergand** [comment]
Hi,
thanks for asking, ~~but I stopped actively working on this project. Maybe a developer of one of the forks can help you out.~~ :)

**Comment by Heistergand** [comment]
Added as request issue: #6

**Comment by arc005ing** [comment]
The keys column count is not showing in 'Show as list', but is working fine in #43 
360 ff is a great fix!

**Reply by Heistergand** [reply]
Thanks for commenting. In case you meant that the keys column count did not show at all when no Keys Plugin was present, this is now fixed in #48 (#47).
I'm not sure what you mean by "360 ff", but take 'great fix' as compliment, thx :)

**Reply by arc005ing** [reply]
360 fanfield, your plugin is great addition to anyones tools!

**Reply by arc005ing** [reply]
There are CORE enabled Inventory Key plugins now that don't require you to update key counts, could they be supported easily? eg https://github.com/IITC-CE/Community-plugins / https://github.com/DanielOndiordna/IngressLiveInventory

**Reply by Heistergand** [reply]
Actually, Fanfields is already in the list of Community-Plugins, but I did not watch out for more 3rd party plugins to support. I'll have a look, but no promises. :)

**Reply by Heistergand** [reply]
Done in #51
